graphql/directive_test.go

```diff
@@ -79,6 +79,15 @@ func setupPermissions(t *testing.T) {
 
 func TestRequireSuperUser(t *testing.T) {
 	setupPermissions(t)
+	dbUser := &user.DBUser{
```

Comment:
Seems like this should clear the user collection before attempting to insert.

Reply:
done